keep/api/api.py

```diff
@@ -35,6 +38,67 @@ async def dispose_context_manager() -> None:
     ContextManager.delete_instance()
 
 
+class EventCaptureMiddleware(BaseHTTPMiddleware):
+    def __init__(self, app: FastAPI):
+        super().__init__(app)
```

Comment:
is this init being called only once when adding the middleware or for every request? if it's for every request, maybe it's better to initalize posthog client once?

Reply:
its once on init